tests/affiliatedcertification/tests/affiliated_certification_container.go

```diff
@@ -121,33 +121,148 @@ var _ = Describe("Affiliated-certification container certification,", func() {
 
 	// 46566
 	It("certifiedcontainerinfo field exists in tnf_config but has no value [skip]", func() {
-		Skip("Under development")
+		By("Add container information to tnf_config.yml")
+		err := globalhelper.DefineTnfConfig(
+			[]string{netparameters.TestNetworkingNameSpace},
+			[]string{netparameters.TestPodLabel},
+			[]string{affiliatedcertparameters.ContainerFieldNoSubFields})
+		Expect(err).ToNot(HaveOccurred(), "Error defining tnf config file")
+
+		By("Start test")
+		err = globalhelper.LaunchTests(
+			[]string{affiliatedcertparameters.AffiliatedCertificationTestSuiteName},
+			affiliatedcertparameters.TestCaseContainerSkipRegEx,
+		)
+		Expect(err).ToNot(HaveOccurred(), "Error running "+
+			affiliatedcertparameters.AffiliatedCertificationTestSuiteName+" test")
+
+		By("Verify test case status in Junit and Claim reports")
+		err = nethelper.ValidateIfReportsAreValid(
+			affiliatedcertparameters.TestCaseContainerAffiliatedCertName,
+			globalparameters.TestCaseSkipped)
+		Expect(err).ToNot(HaveOccurred(), "Error validating test reports")
 	})
 
 	// 46567
 	It("certifiedcontainerinfo field does not exist in tnf_config [skip]", func() {
-		Skip("Under development")
+		By("Add container information to tnf_config.yml")
```

Comment:
It looks like all tests have the same flow. May be we can move the main test flow in to helper function and call it from test cases. What do you think?

Reply:
makes sense to me - will do